runtime/pipeline/publisher.yaml

```diff
@@ -0,0 +1,27 @@
+Description: AWS IAM Role for publishing layers
+
+Parameters:
+  BuilderAccount:
+    Type: String
+
+Resources:
+  LayerPublisherRole:
+    Type: AWS::IAM::Role
+    Properties:
+      RoleName: bref-layer-publisher
+      AssumeRolePolicyDocument:
+        Statement:
+          Effect: Allow
+          Principal:
+            AWS: !Sub "arn:aws:iam::${BuilderAccount}:role/bref-php-binary-builder"
```

Comment:
If this role doesn't already exists in the builder account, this will error out. The reason for that is because IAM will resolve this role uniquely. If the role gets deleted and recreated it still won't work anymore as a protection against permission escalation.